tests/ops/op_math/test_controlled.py

```diff
@@ -842,36 +867,6 @@ def test_no_control_values_no_special_decomp(self, test_expand):
             # pylint: disable=unused-variable
             decomp = op.expand().circuit if test_expand else op.decomposition()
 
-    def test_zyz_decomp_no_control_values(self, test_expand):
-        """Test that the ZYZ decomposition is used for single qubit target operations
-        when other decompositions aren't available."""
-
-        base = qml.RX(0.123, wires="a")
-        op = Controlled(base, (0, 1, 2))
-
-        assert op.has_decomposition
-        decomp = op.expand().circuit if test_expand else op.decomposition()
-        expected = qml.ops.ctrl_decomp_zyz(base, (0, 1, 2))
-        for op, exp_op in zip(decomp, expected):
-            assert qml.equal(op, exp_op)
-
-    def test_zyz_decomp_control_values(self, test_expand):
-        """Test that the ZYZ decomposition is used for single qubit target operations
-        when other decompositions aren't available and control values are present."""
-
-        base = qml.RX(0.123, wires="a")
-        op = Controlled(base, (0, 1, 2), control_values=[True, False, False])
-
-        assert op.has_decomposition
-        decomp = op.expand().circuit if test_expand else op.decomposition()
-        assert qml.equal(qml.PauliX(1), decomp[0])
-        assert qml.equal(qml.PauliX(2), decomp[1])
-        assert qml.equal(qml.PauliX(1), decomp[-2])
-        assert qml.equal(qml.PauliX(2), decomp[-1])
-        expected = qml.ops.ctrl_decomp_zyz(base, (0, 1, 2))
-        for op, exp_op in zip(decomp[2:-2], expected):
-            assert qml.equal(op, exp_op)
-
```

Comment:
Why were these tests removed? Are there still tests for the case where the ZYZ decomposition is used (i.e. when there is a single control wire)?

Reply:
removed because the auto select now uses the bisect algorithm, but I can add back tests with 1 control wire

Reply:
I tried a few with 1 control wire, it seems they all decompose to something else ex. RX -> CRX and it's not using the zyz algorithm

Reply:
with decomposition hints to break the cycle, I got it to work